src/blocks/extensions/advanced-settings/index.js

```diff
@@ -483,12 +483,12 @@ addFilter( 'uag_advance_tab_content', 'uagb/advanced-display-condition', functio
 
 	let notHasDisallowedParentForAnimations = true;
 
-	// Currently we are disallowing animations feature in Tabs block.
+	// Currently we are disallowing animations feature in Tabs & Countdown blocks.
 	if ( getParentBlocks.length ) {
 		for ( let i = 0; i < getParentBlocks.length; i++ ) {
 			const currentParent = wp.data.select( 'core/block-editor' ).getBlock( getParentBlocks[ i ] );
 
-			if ( currentParent.name === 'uagb/tabs' || currentParent.name === 'uagb/tabs-child' ) {
+			if ( currentParent.name === 'uagb/tabs' || currentParent.name === 'uagb/tabs-child' || currentParent.name === 'uagb/countdown' ) {
```

Comment:
Yoda condition, we must use.
Good practice, it is.

```suggestion
			if ( 'uagb/tabs' === currentParent.name || 'uagb/tabs-child' === currentParent.name || 'uagb/countdown' === currentParent.name ) {
```

Reply:
Yoda conditions after your suggestion, used I've now.